pyquil/quil.py

```diff
@@ -73,6 +75,24 @@ def __iter__(self):
         """
         return self.actions.__iter__()
 
+    def get_qubits(self):
```

Comment:
What's the rationale to make this a method on the class? Can we make it a function on pyquil programs?